modules/azure/core/variables.tf

```diff
@@ -24,30 +24,21 @@ variable "vnet_config" {
     address_space = list(string)
     dns_servers   = list(string)
     subnets = list(object({
-      name                                      = string
-      cidr                                      = string
-      service_endpoints                         = list(string)
-      aks_subnet                                = bool
-      private_endpoint_network_policies_enabled = optional(bool, true)
+      name                                          = string
+      cidr                                          = string
+      service_endpoints                             = list(string)
+      aks_subnet                                    = bool
+      private_link_service_network_policies_enabled = optional(bool, true)
+      private_endpoint_network_policies_enabled     = optional(bool, true)
```

Comment:
So at least in AKS these values are already set. And my understanding is that they are also set in the portal by default and in v4 of azurerm the will be true.
So in short true is probably the way to go.

Reply:
My opinion is that unless it needs to be configured it should not have a variable. I think it would be better to set these to true in the module instead.